runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -206,9 +221,6 @@ local function clear(bufnr)
   if bufnr == nil or bufnr == 0 then
```

Comment:
We're not allowing `bufnr` to be nil anymore though right?

Reply:
Thanks, just forgot to remove the judgment here